clouds/aws/aws_roles/aws_roles.py

```diff
@@ -0,0 +1,67 @@
+import botocore.errorfactory
+from aws.aws_clients import IAMClient
+from botocore.exceptions import BotoCoreError
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+
+class AWSRoles:
```

Comment:
no need for a class here; have a function to get the client and then fucntions which get it as arg